driver-core/build.gradle.kts

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2008-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import ProjectExtensions.configureJarManifest
+import ProjectExtensions.configureMavenPublication
+
+plugins {
+    id("project.java")
+    id("conventions.test-artifacts")
+    id("conventions.testing-mockito")
+    id("conventions.testing-spock-exclude-slow")
```

Comment:
so from my understanding, the above will bring in transitively  the plugin `conventions.testing-junit-vintage`, but not `conventions.testing-junit`. But it seems JUnit 5 is used extensively together with Spock (which is based on JUnit 4). Should we add `conventions.testing-junit` as well?

Reply:
Done :)